docs/sources/alerting/_index.md

```diff
@@ -2,6 +2,7 @@
 aliases:
   - about-alerting/
   - unified-alerting/alerting/
```

Comment:
nit: I propose this to be explicit and consistent with others.
```suggestion
  - ./unified-alerting/alerting/
```

Reply:
thank you! will do